codalab_service.py

```diff
@@ -203,7 +209,7 @@ def _get_parser():
                 '--version',
                 '-v',
                 type=str,
-                help='CodaLab version to use for building and deployment',
+                help='CodaLab version to use for building and deployment (should be branch name)',
```

Comment:
Since it's been removed from Server-Setup.md, can we add a note that it's only used for Travis?

Also, what's the default value for this when running locally?

Reply:
Done.  Defaults to the git branch name.